### DISCUSSION TITLE
Section "6.1.2 Information about winners" in visualization template (SDK 1.7)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#679] [Q&A] [answered]
We are generating our eForms with SDK 1.7 and when we generate eForms of subtype 29 (Contract or concession award notice – standard regime), when viewing them in eNotices2 we see the "6.1.2 Information about winners" section empty.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/536998bc-4928-4a4c-b762-bdefba7c51cf)

We have searched for production eForms of the same type (subtype 29, SDK 1.7) sent by other eSenders and we have verified that it also appears empty.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/3e0c990c-3aa7-4209-bdbc-eda4c02efb66)

[https://ted.europa.eu/udl?uri=TED:NOTICE:547708-2023:TEXT:EN:HTML](url)

On the other hand subtypes 30 generated by the same eSender do show information in that section.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/d10c0665-be50-48d8-8e70-18c78c801866)

[https://ted.europa.eu/udl?uri=TED:NOTICE:553546-2023:TEXT:EN:HTML](url)

On the other hand, other eSenders that generate their eForms (subtype 29) with different SDKs do show the information in the visualization.

For example, Estonia, which generates eForms with SDK 1.3, does display that section correctly.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/474c4fad-92bb-4278-ab5b-0565b3eb7f4d)

[https://ted.europa.eu/udl?uri=TED:NOTICE:568220-2023:TEXT:EN:HTML](url)

Switzerland, which generates eForms with SDK 1.6, does display that section correctly.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/229c43fe-9554-4546-ac39-8fc21a0e3b6d)

[https://ted.europa.eu/udl?uri=TED:NOTICE:568217-2023:TEXT:EN:HTML](url)

Belgium, which generates eForms with SDK 1.8, does display that section correctly.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/5c7e93e0-09f4-4379-8846-477caad891c8)

[https://ted.europa.eu/udl?uri=TED:NOTICE:568155-2023:TEXT:EN:HTML](url)

Doesn't that information have to be shown? Are there any display issues with SDK 1.7 and 29 subtypes?

Thanks in advance

**Comment by pdonohoe** [comment, marked as answer]
There is no particular issue with SDK 1.7 and subtype 29 for the display of Winners. The information for Winners is shown if the information included in the notice XML is correct. 

In the example notice you cite where the Winner is not shown (https://ted.europa.eu/udl?uri=TED:NOTICE:547708-2023:TEXT:EN:HTML), the XML is not correct. Firstly, it contains only 3 Organisations: the Contracting Authority (ORG-0001, "Rajavartiolaitos"), the eSender (ORG-0002, "Hansel Oy") and the Review Info Provider and Review Organization (ORG-0003, "Markkinaoikeus"). It does not include an Organisation corresponding to any Tenderers or Winners. Secondly, while the LotResult identifies the associated Tender and Contract, and the Tender identifies the associated Tendering Party (TPA-0001), the TenderingParty does not identify any component Organisations. If I insert a reference (OPT-300-Tenderer) to ORG-0003, then the Winner section is populated with the information for that Organisation.

In your second example (https://ted.europa.eu/udl?uri=TED:NOTICE:553546-2023:TEXT:EN:HTML), the LotResult references three LotTenders, each of which is referenced by a Contract and references a Tender, each of which reference a TenderingParty, each of which references an Organisation. Thus there are three winners shown.

The EFX code responsible for displaying this section follows this logic:

For a LotResult, read the included Tender Identifier References (OPT-320-LotResult). These are filtered to include only those that are also referenced by a SettledContract (BT-3202-Contract : "Contract Tender ID (Reference)"). For each of the identified Tenders (OPT-321-Tender : "Tender Technical Identifier"), a list of TenderingParty items is read (OPT-310-Tender : "Tendering Party ID Reference"). For each TenderingParty item referenced (OPT-210-Tenderer : "Tendering Party ID"), a list of Tendering Organisations is read (OPT-300-Tenderer : "Tenderer ID Reference"). Each of these Organisations (OPT-200-Organization-Company : "Organisation Technical Identifier") is displayed as a Winner.

You will need to ensure that all of these business terms are present for a Winner to be displayed.

**Reply by AlexAS81** [reply]
We are reviewing the type 29 eForms that we are generating and we do not see where the problem is. Could you tell us what is not correct?

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ContractAwardNotice xmlns="urn:oasis:names:specification:ubl:schema:xsd:ContractAwardNotice-2"
                     xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2"
                     xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
                     xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2"
                     xmlns:efac="http://data.europa.eu/p27/eforms-ubl-extension-aggregate-components/1"
                     xmlns:efbc="http://data.europa.eu/p27/eforms-ubl-extension-basic-components/1"
                     xmlns:efext="http://data.europa.eu/p27/eforms-ubl-extensions/1">
   <ext:UBLExtensions>
      <ext:UBLExtension>
         <ext:ExtensionContent>
            <efext:EformsExtension>
               <efac:NoticeResult>
                  <efac:LotResult>
                     <cbc:ID>RES-0001</cbc:ID>
                     <cbc:TenderResultCode listName="winner-selection-status">selec-w</cbc:TenderResultCode>
                     <efac:LotTender>
                        <cbc:ID>TEN-0001</cbc:ID>
                     </efac:LotTender>
                     <efac:SettledContract>
                        <cbc:ID>CON-0001</cbc:ID>
                     </efac:SettledContract>
                     <efac:TenderLot>
                        <cbc:ID>LOT-0001</cbc:ID>
                     </efac:TenderLot>
                  </efac:LotResult>
                  <efac:LotTender>
                     <cbc:ID>TEN-0001</cbc:ID>
                     <efac:TenderingParty>
                        <cbc:ID>TPA-0001</cbc:ID>
                     </efac:TenderingParty>
                     <efac:TenderLot>
                        <cbc:ID>LOT-0001</cbc:ID>
                     </efac:TenderLot>
                  </efac:LotTender>
                  <efac:SettledContract>
                     <cbc:ID>CON-0001</cbc:ID>
                     <efac:LotTender>
                        <cbc:ID>TEN-0001</cbc:ID>
                     </efac:LotTender>
                  </efac:SettledContract>
                  <efac:TenderingParty>
                     <cbc:ID>TPA-0001</cbc:ID>
                     <efac:Tenderer>
                        <cbc:ID>ORG-0005</cbc:ID>
                     </efac:Tenderer>
                  </efac:TenderingParty>
               </efac:NoticeResult>
               <efac:NoticeSubType>
                  <cbc:SubTypeCode listName="notice-subtype">29</cbc:SubTypeCode>
               </efac:NoticeSubType>
               <efac:Organizations>
                  <efac:Organization>
                     <efac:Company>
                        <cbc:WebsiteURI>http://www.bizkaia.eus</cbc:WebsiteURI>
                        <cac:PartyIdentification>
                           <cbc:ID>ORG-0001</cbc:ID>
                        </cac:PartyIdentification>
                        <cac:PartyName>
                           <cbc:Name languageID="SPA">Tribunal administrativo foral de recursos contractuales de Bizkaia</cbc:Name>
                        </cac:PartyName>
                        <cac:PostalAddress>
                           <cbc:StreetName>Gran Vía Diego López de Haro, 25 (Entreplanta-Asesoría Jurídica)</cbc:StreetName>
                           <cbc:CityName>Bilbao</cbc:CityName>
                           <cbc:PostalZone>48009</cbc:PostalZone>
                           <cac:Country>
                              <cbc:IdentificationCode listName="country">ESP</cbc:IdentificationCode>
                           </cac:Country>
                        </cac:PostalAddress>
                        <cac:PartyLegalEntity>
                           <cbc:CompanyID>57187.0</cbc:CompanyID>
                        </cac:PartyLegalEntity>
                        <cac:Contact>
                           <cbc:Telephone>+34 944068000</cbc:Telephone>
                           <cbc:ElectronicMail>tribunal.recursos.contractuales@bizkaia.eus</cbc:ElectronicMail>
                        </cac:Contact>
                     </efac:Company>
                  </efac:Organization>
                  <efac:Organization>
                     <efac:Company>
                        <cbc:WebsiteURI>http://www.google.es</cbc:WebsiteURI>
                        <cac:PartyIdentification>
                           <cbc:ID>ORG-0003</cbc:ID>
                        </cac:PartyIdentification>
                        <cac:PartyName>
                           <cbc:Name languageID="SPA">Gobierno Vasco - Hacienda y Economía</cbc:Name>
                        </cac:PartyName>
                        <cac:PostalAddress>
                           <cbc:StreetName>Contacto de la entidad </cbc:StreetName>
                           <cbc:CityName>Vitoria-Gasteiz</cbc:CityName>
                           <cbc:PostalZone>12345</cbc:PostalZone>
                           <cac:Country>
                              <cbc:IdentificationCode listName="country">ESP</cbc:IdentificationCode>
                           </cac:Country>
                        </cac:PostalAddress>
                        <cac:PartyLegalEntity>
                           <cbc:CompanyID>S4833001C</cbc:CompanyID>
                        </cac:PartyLegalEntity>
                        <cac:Contact>
                           <cbc:Name>saas</cbc:Name>
                           <cbc:Telephone>+34 945555555</cbc:Telephone>
                           <cbc:ElectronicMail>email@entidad.com</cbc:ElectronicMail>
                        </cac:Contact>
                     </efac:Company>
                  </efac:Organization>
                  <efac:Organization>
                     <efac:Company>
                        <cac:PartyIdentification>
                           <cbc:ID>ORG-0005</cbc:ID>
                        </cac:PartyIdentification>
                        <cac:PartyName>
                           <cbc:Name languageID="SPA">SERIKAT CONSULTORÍA E INFORMÁTICA, S.A.</cbc:Name>
                        </cac:PartyName>
                        <cac:PostalAddress>
                           <cbc:StreetName>Ercilla, 19, 6ª Planta</cbc:StreetName>
                           <cbc:CityName>Bilbao</cbc:CityName>
                           <cbc:PostalZone>48009</cbc:PostalZone>
                           <cac:Country>
                              <cbc:IdentificationCode listName="country">ESP</cbc:IdentificationCode>
                           </cac:Country>
                        </cac:PostalAddress>
                        <cac:PartyLegalEntity>
                           <cbc:CompanyID>S4833001C</cbc:CompanyID>
                        </cac:PartyLegalEntity>
                        <cac:Contact>
                           <cbc:Telephone>+34 944792030</cbc:Telephone>
                           <cbc:ElectronicMail>e-concursos@serikat.es</cbc:ElectronicMail>
                        </cac:Contact>
                     </efac:Company>
                  </efac:Organization>
               </efac:Organizations>
            </efext:EformsExtension>
         </ext:ExtensionContent>
      </ext:UBLExtension>
   </ext:UBLExtensions>
   <cbc:UBLVersionID>2.3</cbc:UBLVersionID>
   <cbc:CustomizationID>eforms-sdk-1.7</cbc:CustomizationID>
   <cbc:ID schemeName="notice-id">27ecffa0-3cba-4828-a75b-f87d0b9aa9oc</cbc:ID>
   <cbc:ContractFolderID>4bce2c84-03ee-45a2-9c99-16da989426ed</cbc:ContractFolderID>
   <cbc:IssueDate>2023-09-22+02:00</cbc:IssueDate>
   <cbc:IssueTime>11:17:04+02:00</cbc:IssueTime>
   <cbc:VersionID>01</cbc:VersionID>
   <cbc:RegulatoryDomain>32014L0024</cbc:RegulatoryDomain>
   <cbc:NoticeTypeCode listName="result">can-standard</cbc:NoticeTypeCode>
   <cbc:NoticeLanguageCode listName="language">SPA</cbc:NoticeLanguageCode>
   <cac:ContractingParty>
      <cbc:BuyerProfileURI>https://www.contratacion.euskadi.eus/</cbc:BuyerProfileURI>
      <cac:ContractingPartyType>
         <cbc:PartyTypeCode listName="buyer-legal-type">ra</cbc:PartyTypeCode>
      </cac:ContractingPartyType>
      <cac:ContractingActivity>
         <cbc:ActivityTypeCode listName="authority-activity">gen-pub</cbc:ActivityTypeCode>
      </cac:ContractingActivity>
      <cac:Party>
         <cac:PartyIdentification>
            <cbc:ID>ORG-0003</cbc:ID>
         </cac:PartyIdentification>
      </cac:Party>
   </cac:ContractingParty>
   <cac:TenderingProcess>
      <cbc:Description languageID="SPA">Abierto</cbc:Description>
      <cbc:ProcedureCode listName="procurement-procedure-type">open</cbc:ProcedureCode>
      <cac:ProcessJustification>
         <cbc:ProcessReasonCode listName="accelerated-procedure">false</cbc:ProcessReasonCode>
      </cac:ProcessJustification>
   </cac:TenderingProcess>
   <cac:ProcurementProject>
      <cbc:ID>Objeto del contrato en castellano</cbc:ID>
      <cbc:Name languageID="SPA">Objeto del contrato en castellano</cbc:Name>
      <cbc:Description languageID="SPA">Objeto del contrato en castellano</cbc:Description>
      <cbc:ProcurementTypeCode listName="contract-nature">works</cbc:ProcurementTypeCode>
      <cac:RequestedTenderTotal>
         <cbc:EstimatedOverallContractAmount currencyID="EUR">35235</cbc:EstimatedOverallContractAmount>
      </cac:RequestedTenderTotal>
      <cac:MainCommodityClassification>
         <cbc:ItemClassificationCode listName="cpv">45000000</cbc:ItemClassificationCode>
      </cac:MainCommodityClassification>
      <cac:RealizedLocation>
         <cac:Address>
            <cbc:CountrySubentityCode listName="nuts">ES211</cbc:CountrySubentityCode>
         </cac:Address>
      </cac:RealizedLocation>
   </cac:ProcurementProject>
   <cac:ProcurementProjectLot>
      <cbc:ID schemeName="Lot">LOT-0001</cbc:ID>
      <cac:TenderingTerms>
         <cbc:FundingProgramCode listName="eu-funded">no-eu-funds</cbc:FundingProgramCode>
         <cac:AwardingTerms>
            <cac:AwardingCriterion>
               <cac:SubordinateAwardingCriterion>
                  <cbc:AwardingCriterionTypeCode listName="award-criterion-type">price</cbc:AwardingCriterionTypeCode>
                  <cbc:Name languageID="SPA">Mejor precio</cbc:Name>
               </cac:SubordinateAwardingCriterion>
            </cac:AwardingCriterion>
         </cac:AwardingTerms>
         <cac:AdditionalInformationParty>
            <cac:PartyIdentification>
               <cbc:ID>ORG-0003</cbc:ID>
            </cac:PartyIdentification>
         </cac:AdditionalInformationParty>
         <cac:AppealTerms>
            <cac:AppealReceiverParty>
               <cac:PartyIdentification>
                  <cbc:ID>ORG-0001</cbc:ID>
               </cac:PartyIdentification>
            </cac:AppealReceiverParty>
         </cac:AppealTerms>
      </cac:TenderingTerms>
      <cac:TenderingProcess>
         <cbc:GovernmentAgreementConstraintIndicator>false</cbc:GovernmentAgreementConstraintIndicator>
         <cac:AuctionTerms>
            <cbc:AuctionConstraintIndicator>false</cbc:AuctionConstraintIndicator>
         </cac:AuctionTerms>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="framework-agreement">none</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="dps-usage">none</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
      </cac:TenderingProcess>
      <cac:ProcurementProject>
         <cbc:ID>Objeto del contrato en castellano</cbc:ID>
         <cbc:Name languageID="SPA">Objeto del contrato en castellano</cbc:Name>
         <cbc:Description languageID="SPA">Objeto del contrato en castellano</cbc:Description>
         <cbc:ProcurementTypeCode listName="contract-nature">works</cbc:ProcurementTypeCode>
         <cac:MainCommodityClassification>
            <cbc:ItemClassificationCode listName="cpv">45000000</cbc:ItemClassificationCode>
         </cac:MainCommodityClassification>
         <cac:AdditionalCommodityClassification>
            <cbc:ItemClassificationCode listName="cpv">45000000</cbc:ItemClassificationCode>
         </cac:AdditionalCommodityClassification>
         <cac:PlannedPeriod>
            <cbc:DurationMeasure unitCode="DAY">12</cbc:DurationMeasure>
         </cac:PlannedPeriod>
      </cac:ProcurementProject>
   </cac:ProcurementProjectLot>
   <cac:TenderResult>
      <cbc:AwardDate>2000-01-01+01:00</cbc:AwardDate>
   </cac:TenderResult>
</ContractAwardNotice>
```

Thanks in advance

**Reply by bertrand-lorentz** [reply]
For this XML, the only 2 errors reported by CVS are:

1/ The format of the notice identifier is incorrect (it contains the letter 'o' which is not allowed in a UUID)
```xml
<svrl:failed-assert id="BR-BT-00701-0052" location="/can:ContractAwardNotice/cbc:ID" test="matches(normalize-space(.),'^[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89ab][a-f0-9]{3}-[a-f0-9]{12}$')" role="ERROR">
    <svrl:text>BT-701-notice must match the following pattern: ProcedureOrNoticeID</svrl:text>
</svrl:failed-assert>
```

2/ The dispatch date is now too old, this is expected as time as passed since you created this notice
```xml
<svrl:failed-assert location="/can:ContractAwardNotice/cbc:IssueDate" test="((current-date() - xs:date(text())) le xs:dayTimeDuration('P2D')) and ((current-date() - xs:date(text())) ge xs:dayTimeDuration('-P1D'))" role="ERROR">
    <svrl:text>Notice Dispatch Date must be between 0 and 24 hours before the current date.</svrl:text>
</svrl:failed-assert>
```

**Reply by AlexAS81** [reply]
We modify the notice identifier by hand to do a quick test.
We will immediately send you an example with today's date.

**Reply by AlexAS81** [reply]
Can you please tell us what is not correct in this XML so that section 6.1.2 is not shown in the display template?

`<?xml version="1.0" encoding="UTF-8"?>
<ContractAwardNotice xmlns="urn:oasis:names:specification:ubl:schema:xsd:ContractAwardNotice-2"
                     xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2"
                     xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
                     xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2"
                     xmlns:efac="http://data.europa.eu/p27/eforms-ubl-extension-aggregate-components/1"
                     xmlns:efbc="http://data.europa.eu/p27/eforms-ubl-extension-basic-components/1"
                     xmlns:efext="http://data.europa.eu/p27/eforms-ubl-extensions/1">
   <ext:UBLExtensions>
      <ext:UBLExtension>
         <ext:ExtensionContent>
            <efext:EformsExtension>
               <efac:NoticeResult>
                  <efac:LotResult>
                     <cbc:ID>RES-0001</cbc:ID>
                     <cbc:TenderResultCode listName="winner-selection-status">selec-w</cbc:TenderResultCode>
                     <efac:LotTender>
                        <cbc:ID>TEN-0001</cbc:ID>
                     </efac:LotTender>
                     <efac:SettledContract>
                        <cbc:ID>CON-0001</cbc:ID>
                     </efac:SettledContract>
                     <efac:TenderLot>
                        <cbc:ID>LOT-0001</cbc:ID>
                     </efac:TenderLot>
                  </efac:LotResult>
                  <efac:LotTender>
                     <cbc:ID>TEN-0001</cbc:ID>
                     <efac:TenderingParty>
                        <cbc:ID>TPA-0001</cbc:ID>
                     </efac:TenderingParty>
                     <efac:TenderLot>
                        <cbc:ID>LOT-0001</cbc:ID>
                     </efac:TenderLot>
                  </efac:LotTender>
                  <efac:SettledContract>
                     <cbc:ID>CON-0001</cbc:ID>
                     <efac:LotTender>
                        <cbc:ID>TEN-0001</cbc:ID>
                     </efac:LotTender>
                  </efac:SettledContract>
                  <efac:TenderingParty>
                     <cbc:ID>TPA-0001</cbc:ID>
                     <efac:Tenderer>
                        <cbc:ID>ORG-0005</cbc:ID>
                     </efac:Tenderer>
                  </efac:TenderingParty>
               </efac:NoticeResult>
               <efac:NoticeSubType>
                  <cbc:SubTypeCode listName="notice-subtype">29</cbc:SubTypeCode>
               </efac:NoticeSubType>
               <efac:Organizations>
                  <efac:Organization>
                     <efac:Company>
                        <cbc:WebsiteURI>http://www.bizkaia.eus</cbc:WebsiteURI>
                        <cac:PartyIdentification>
                           <cbc:ID>ORG-0001</cbc:ID>
                        </cac:PartyIdentification>
                        <cac:PartyName>
                           <cbc:Name languageID="SPA">Tribunal administrativo foral de recursos contractuales de Bizkaia</cbc:Name>
                        </cac:PartyName>
                        <cac:PostalAddress>
                           <cbc:StreetName>Gran Vía Diego López de Haro, 25 (Entreplanta-Asesoría Jurídica)</cbc:StreetName>
                           <cbc:CityName>Bilbao</cbc:CityName>
                           <cbc:PostalZone>48009</cbc:PostalZone>
                           <cac:Country>
                              <cbc:IdentificationCode listName="country">ESP</cbc:IdentificationCode>
                           </cac:Country>
                        </cac:PostalAddress>
                        <cac:PartyLegalEntity>
                           <cbc:CompanyID>57187.0</cbc:CompanyID>
                        </cac:PartyLegalEntity>
                        <cac:Contact>
                           <cbc:Telephone>+34 944068000</cbc:Telephone>
                           <cbc:ElectronicMail>tribunal.recursos.contractuales@bizkaia.eus</cbc:ElectronicMail>
                        </cac:Contact>
                     </efac:Company>
                  </efac:Organization>
                  <efac:Organization>
                     <efac:Company>
                        <cbc:WebsiteURI>http://www.google.es</cbc:WebsiteURI>
                        <cac:PartyIdentification>
                           <cbc:ID>ORG-0003</cbc:ID>
                        </cac:PartyIdentification>
                        <cac:PartyName>
                           <cbc:Name languageID="SPA">Gobierno Vasco - Hacienda y Economía</cbc:Name>
                        </cac:PartyName>
                        <cac:PostalAddress>
                           <cbc:StreetName>Contacto de la entidad </cbc:StreetName>
                           <cbc:CityName>Vitoria-Gasteiz</cbc:CityName>
                           <cbc:PostalZone>12345</cbc:PostalZone>
                           <cac:Country>
                              <cbc:IdentificationCode listName="country">ESP</cbc:IdentificationCode>
                           </cac:Country>
                        </cac:PostalAddress>
                        <cac:PartyLegalEntity>
                           <cbc:CompanyID>S4833001C</cbc:CompanyID>
                        </cac:PartyLegalEntity>
                        <cac:Contact>
                           <cbc:Name>saas</cbc:Name>
                           <cbc:Telephone>+34 945555555</cbc:Telephone>
                           <cbc:ElectronicMail>email@entidad.com</cbc:ElectronicMail>
                        </cac:Contact>
                     </efac:Company>
                  </efac:Organization>
                  <efac:Organization>
                     <efac:Company>
                        <cac:PartyIdentification>
                           <cbc:ID>ORG-0005</cbc:ID>
                        </cac:PartyIdentification>
                        <cac:PartyName>
                           <cbc:Name languageID="SPA">SERIKAT CONSULTORÍA E INFORMÁTICA, S.A.</cbc:Name>
                        </cac:PartyName>
                        <cac:PostalAddress>
                           <cbc:StreetName>Ercilla, 19, 6ª Planta</cbc:StreetName>
                           <cbc:CityName>Bilbao</cbc:CityName>
                           <cbc:PostalZone>48009</cbc:PostalZone>
                           <cac:Country>
                              <cbc:IdentificationCode listName="country">ESP</cbc:IdentificationCode>
                           </cac:Country>
                        </cac:PostalAddress>
                        <cac:PartyLegalEntity>
                           <cbc:CompanyID>S4833001C</cbc:CompanyID>
                        </cac:PartyLegalEntity>
                        <cac:Contact>
                           <cbc:Telephone>+34 944792030</cbc:Telephone>
                           <cbc:ElectronicMail>e-concursos@serikat.es</cbc:ElectronicMail>
                        </cac:Contact>
                     </efac:Company>
                  </efac:Organization>
               </efac:Organizations>
            </efext:EformsExtension>
         </ext:ExtensionContent>
      </ext:UBLExtension>
   </ext:UBLExtensions>
   <cbc:UBLVersionID>2.3</cbc:UBLVersionID>
   <cbc:CustomizationID>eforms-sdk-1.7</cbc:CustomizationID>
   <cbc:ID schemeName="notice-id">d6ca4056-2ef7-416b-9af7-cc78523ab29a</cbc:ID>
   <cbc:ContractFolderID>e0973425-3095-4627-a334-45af6043f5f8</cbc:ContractFolderID>
   <cbc:IssueDate>2023-09-26+02:00</cbc:IssueDate>
   <cbc:IssueTime>11:00:53+02:00</cbc:IssueTime>
   <cbc:VersionID>01</cbc:VersionID>
   <cbc:RegulatoryDomain>32014L0024</cbc:RegulatoryDomain>
   <cbc:NoticeTypeCode listName="result">can-standard</cbc:NoticeTypeCode>
   <cbc:NoticeLanguageCode listName="language">SPA</cbc:NoticeLanguageCode>
   <cac:ContractingParty>
      <cbc:BuyerProfileURI>https://www.contratacion.euskadi.eus/</cbc:BuyerProfileURI>
      <cac:ContractingPartyType>
         <cbc:PartyTypeCode listName="buyer-legal-type">ra</cbc:PartyTypeCode>
      </cac:ContractingPartyType>
      <cac:ContractingActivity>
         <cbc:ActivityTypeCode listName="authority-activity">gen-pub</cbc:ActivityTypeCode>
      </cac:ContractingActivity>
      <cac:Party>
         <cac:PartyIdentification>
            <cbc:ID>ORG-0003</cbc:ID>
         </cac:PartyIdentification>
      </cac:Party>
   </cac:ContractingParty>
   <cac:TenderingProcess>
      <cbc:Description languageID="SPA">Abierto</cbc:Description>
      <cbc:ProcedureCode listName="procurement-procedure-type">open</cbc:ProcedureCode>
      <cac:ProcessJustification>
         <cbc:ProcessReasonCode listName="accelerated-procedure">false</cbc:ProcessReasonCode>
      </cac:ProcessJustification>
   </cac:TenderingProcess>
   <cac:ProcurementProject>
      <cbc:ID>Objeto del contrato en castellano</cbc:ID>
      <cbc:Name languageID="SPA">Objeto del contrato en castellano</cbc:Name>
      <cbc:Description languageID="SPA">Objeto del contrato en castellano</cbc:Description>
      <cbc:ProcurementTypeCode listName="contract-nature">works</cbc:ProcurementTypeCode>
      <cac:RequestedTenderTotal>
         <cbc:EstimatedOverallContractAmount currencyID="EUR">35235</cbc:EstimatedOverallContractAmount>
      </cac:RequestedTenderTotal>
      <cac:MainCommodityClassification>
         <cbc:ItemClassificationCode listName="cpv">45000000</cbc:ItemClassificationCode>
      </cac:MainCommodityClassification>
      <cac:RealizedLocation>
         <cac:Address>
            <cbc:CountrySubentityCode listName="nuts">ES211</cbc:CountrySubentityCode>
         </cac:Address>
      </cac:RealizedLocation>
   </cac:ProcurementProject>
   <cac:ProcurementProjectLot>
      <cbc:ID schemeName="Lot">LOT-0001</cbc:ID>
      <cac:TenderingTerms>
         <cbc:FundingProgramCode listName="eu-funded">no-eu-funds</cbc:FundingProgramCode>
         <cac:AwardingTerms>
            <cac:AwardingCriterion>
               <cac:SubordinateAwardingCriterion>
                  <cbc:AwardingCriterionTypeCode listName="award-criterion-type">price</cbc:AwardingCriterionTypeCode>
                  <cbc:Name languageID="SPA">Mejor precio</cbc:Name>
               </cac:SubordinateAwardingCriterion>
            </cac:AwardingCriterion>
         </cac:AwardingTerms>
         <cac:AdditionalInformationParty>
            <cac:PartyIdentification>
               <cbc:ID>ORG-0003</cbc:ID>
            </cac:PartyIdentification>
         </cac:AdditionalInformationParty>
         <cac:AppealTerms>
            <cac:AppealReceiverParty>
               <cac:PartyIdentification>
                  <cbc:ID>ORG-0001</cbc:ID>
               </cac:PartyIdentification>
            </cac:AppealReceiverParty>
         </cac:AppealTerms>
      </cac:TenderingTerms>
      <cac:TenderingProcess>
         <cbc:GovernmentAgreementConstraintIndicator>false</cbc:GovernmentAgreementConstraintIndicator>
         <cac:AuctionTerms>
            <cbc:AuctionConstraintIndicator>false</cbc:AuctionConstraintIndicator>
         </cac:AuctionTerms>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="framework-agreement">none</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="dps-usage">none</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
      </cac:TenderingProcess>
      <cac:ProcurementProject>
         <cbc:ID>Objeto del contrato en castellano</cbc:ID>
         <cbc:Name languageID="SPA">Objeto del contrato en castellano</cbc:Name>
         <cbc:Description languageID="SPA">Objeto del contrato en castellano</cbc:Description>
         <cbc:ProcurementTypeCode listName="contract-nature">works</cbc:ProcurementTypeCode>
         <cac:MainCommodityClassification>
            <cbc:ItemClassificationCode listName="cpv">45000000</cbc:ItemClassificationCode>
         </cac:MainCommodityClassification>
         <cac:AdditionalCommodityClassification>
            <cbc:ItemClassificationCode listName="cpv">45000000</cbc:ItemClassificationCode>
         </cac:AdditionalCommodityClassification>
         <cac:PlannedPeriod>
            <cbc:DurationMeasure unitCode="DAY">12</cbc:DurationMeasure>
         </cac:PlannedPeriod>
      </cac:ProcurementProject>
   </cac:ProcurementProjectLot>
   <cac:TenderResult>
      <cbc:AwardDate>2000-01-01+01:00</cbc:AwardDate>
   </cac:TenderResult>
</ContractAwardNotice>`

This is what it looks like in the PDF:

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/1447ced9-fbcc-4fc8-9d7b-f1fbfbea955c)

**Comment by pdonohoe** [comment]
The XML posted above looks correct.
The PDF generated by the current TED Viewer API shows the Winner:
![image](https://github.com/OP-TED/eForms-SDK/assets/1267429/b1e41c04-cf96-4149-9b02-295b06f5f287)

**Reply by AlexAS81** [reply]
We just tried rendering the XML using the TED Viewer API and it worked correctly.

So there is a problem in displaying the PDFs of the eForms in eNotices2 Web UI.

**Comment by pdonohoe** [comment]
The output of eNotices2 is not intended to reflect the correct format of notices submitted via API. The TED Viewer API may render a notice closer to the rendering when it is published.
